src/main/webapp/index.html

```diff
@@ -96,6 +76,8 @@
 
       <input type="submit" />
     </form>
+    <div id="class-id"></div>
+    <button onclick="setParam('term', 'Spring 2020')">Add parameter</button>
```

Comment:
is there a reason we are hard coding this to "('term', 'Spring 2020')?

Reply:
It was for my own testing and forgot to delete it, figuring that out now!